src/main/java/org/folio/orders/rest/exceptions/HttpException.java

```diff
@@ -2,7 +2,7 @@
 
 import org.folio.orders.utils.ErrorCodes;
 
-public class HttpException extends Exception {
+public class HttpException extends RuntimeException {
```

Comment:
I'm curious why this needs to be an unchecked exception?

Reply:
Still curious, but accepted the PR since this isn't important enough to hold this up.

Reply:
I made the exception unchecked for the following reasons:

- The checked exception forces us to add try/catch or throws the exception , even if it has already been handled in `exceptionaly()`.
- Before this pr, in all cases of using this exception, we wrapped it in an unchecked CompletionException.
- All exceptions, including this one, are processed inside the module, and potential users of our API do not have to handle it.